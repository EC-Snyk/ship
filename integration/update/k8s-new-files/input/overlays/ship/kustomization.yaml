kind: ""
apiversion: ""
patchesStrategicMerge:
- redis-master-service.yaml
- redis-slave-service.yaml
bases:
- ../defaults
patchesJson6902:
- target:
    group: apps
    version: v1
    kind: Deployment
    name: redis-slave
  path: container-name-patch.json
