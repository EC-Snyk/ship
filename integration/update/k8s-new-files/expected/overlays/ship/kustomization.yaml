kind: ""
apiversion: ""
patchesStrategicMerge:
- redis-master-service.yaml
- redis-slave-service.yaml
patchesJson6902:
- target:
    group: apps
    version: v1
    kind: Deployment
    name: redis-slave
  path: container-name-patch.json
bases:
- ../defaults
